pom.xml

```diff
@@ -173,6 +173,7 @@
                         <DynamicImport-Package>
                             *
                         </DynamicImport-Package>
+                        <_noee>true</_noee>
```

Comment:
while this is going to work, the right fix is to fix the build (as was done in master) so the maven-bundle-plugin can find and set correct capability in the manifest

Reply:
I don't remember now the exact line of code from bndlib that generates the Require-Capability, but I know it comes from maven-compiler settings. And while JAF needs JDK9+ for module-info.java, I think it'll stay at EE (execution environment) requirement of JDK9...

May I ask what are you referring to in `as was done in master`?

Reply:
> And while JAF needs JDK9+ for module-info.java, I think it'll stay at EE (execution environment) requirement of JDK9...

Not true. There was some fix on the bnd lib side which - in short - ignores module-info when the value for EE is being generated

> May I ask what are you referring to in as was done in master

to changes which were done after 1.2.2 got released